Engine/gapi/directx12/dxcommandbuffer.cpp

```diff
@@ -590,6 +590,31 @@ void DxCommandBuffer::dispatch(size_t x, size_t y, size_t z) {
   impl->Dispatch(UINT(x),UINT(y),UINT(z));
   }
 
+void DxCommandBuffer::dispatchIndirect(const AbstractGraphicsApi::Buffer& indirect, size_t offset) {
+  const DxBuffer& ind = reinterpret_cast<const DxBuffer&>(indirect);
+  auto& sign = dev.dispatchIndirectSgn.get();
+
+  curUniforms->ssboBarriers(resState, PipelineStage::S_Compute);
+
+  // block future writers
+  resState.onUavUsage(ind.nonUniqId, NonUniqResId::I_None, PipelineStage::S_Compute);
+  resState.flush(*this);
+
+  if (indirectCmd.find(ind.impl.get()) == indirectCmd.end()) {
+    indirectCmd.insert(ind.impl.get());
```

Comment:
This code make sense only for graphics.
In draw/mesh idea is to transition buffer into `D3D12_RESOURCE_STATE_INDIRECT_ARGUMENT` and then transition them back after render-pass ended.

Since compute is outside of render-pass, that wont work.